### PR TITLE
Update release notes

### PR DESCRIPTION
**Pull request by fitzthum**
What else needs to be added? 

@dcmiddle any enclave-cc stuff?

### CODE REVIEW

**Comment by fidencio** [comment]
I'd also add:

* Measured rootfs now works with kata-clh, kata-qemu, kata-clh-tdx, and kata-qemu-tdx
* kata-qemu-tdx supports and has been tested with Verdictd (@arronwy, wants to expand on this?)

**Comment by fidencio** [comment]
@BbolroC, wants to add something related to s390x?

**Comment by BbolroC** [comment]
> @BbolroC, wants to add something related to s390x?

@hbrueckner could give you decent words. Thanks.

**Comment by sameo** [comment]
> What else needs to be added?
> 
> @dcmiddle any enclave-cc stuff?

Adding @mythi for the enclave-cc, as I think it's worth mentioning it in the release notes.

**Comment by sameo** [comment]
>     * kata-qemu-tdx supports and has been tested with Verdictd (@arronwy, do you want to expand on this?)

That's interesting indeed. What were the related CoCo changes?

File: releases/v0.3.0.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+Code Freeze: 1/13/23
 
 Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential Containers
 
 ## What's new
 - Support for pulling images from authenticated container registries. See [design info](https://github.com/confidential-containers/image-rs/blob/main/docs/image_auth.md).
+- Attestation support for AMD SEV-ES
+- Support for `get_resource` endpoint with SEV(-ES)
+- Significantly reduced resource requirements for image pulling
+- SEV attestation bug fixes
```
Comment (dcmiddle) [inline releases/v0.3.0.md:13]:
```suggestion
- SEV attestation bug fixes
- Enabled cosign signature support in enclave-cc / SGX
- Enhanced docs, config, CI pipeline and test coverage for enclave-cc / SGX 
```

**Comment by dcmiddle** [comment]
@fitzthum could you also please bump the Security Badge from 46% to 49%? (near the end of the file) The comments on remaining work are still valid. Thanks!

**Comment by fitzthum** [comment]
Ok, thanks for the additions everyone. I will update shortly.

**Comment by fitzthum** [comment]
Should have everyone's suggestions now.

**Review by sameo — APPROVED** [review]

**Review by dcmiddle — APPROVED** [review]

**Review by fidencio — APPROVED** [review]
lgtm, thanks @fitzthum!

File: releases/v0.3.0.md [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
 ## What's new
 - Support for pulling images from authenticated container registries. See [design info](https://github.com/confidential-containers/image-rs/blob/main/docs/image_auth.md).
+- Significantly reduced resource requirements for image pulling
+- Attestation support for AMD SEV-ES
+- `kata-qemu-tdx` supports and has been tested with Verdictd
+- Support for `get_resource` endpoint with SEV(-ES)
+- Enabled cosign signature support in enclave-cc / SGX
+- SEV attestation bug fixes
+- Measured rootfs now works with `kata-clh`, `kata-qemu`, `kata-clh-tdx`, and `kata-qemu-tdx` runtime classes.
+- Enhanced docs, config, CI pipeline and test coverage for enclave-cc / SGX
```
Comment (hbrueckner) [inline releases/v0.3.0.md:17]: @fitzthum One more remark on s390x: Could you add this to the `What's new`:

`IBM zSystems / LinuxONE (s390x) enablement and CI verification on non-TEE environments`

Also in section `Hardware Support`, could you replace `IBM Z` with:

`IBM Secure Execution (SE) on IBM zSystems & LinuxONE`

File: releases/v0.3.0.md [hunk 3]
```diff
@@ -1,12 +1,20 @@
 # Release Notes for v0.3.0
-Release Date: TBD
+Release Date: 1/19/23
```
[the remark this answers is not in the document]
Comment (fitzthum) [reply releases/v0.3.0.md]: I have attempted to compromise on the date format.

**Pull request merged**